fix: updated and consolidated quotes

rdm100 opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #38200 
Closes #39072
<!-- Feel free to add any additional description of changes below this line -->

raisedadead reviewed (changes requested):
![image](https://user-images.githubusercontent.com/1884376/84573959-43711800-adc1-11ea-92b7-ef51e1f647cd.png)

We have a dedicated document on what is the best practice for opening PRs.

Again with no disrespect, since this is a fresh PR, please squash all the commits to one to save everyone's time (this was the reason we had to close the original PR in the first place).

rdm100 commented:
@raisedadead I have squashed the commits down to 1. The code is the same, but the Travis build tests have failed even though they passed before I squashed the commits.

How do I fix this?

rdm100 commented:
@raisedadead fixed it, the first time I squashed the commits the last commit was left out, so I made the commit again then squashed the 2 commits into 1.

ojeytonwilliams commented on client/src/components/FourOhFour/index.js:
```diff
@@ -4,7 +4,7 @@ import Spinner from 'react-spinkit';
 
 import notFoundLogo from '../../assets/images/freeCodeCamp-404.svg';
-import { quotes } from '../../resources/quotes.json';
+import words from '../../../../config/motivationalQuotes.json';
```
```suggestion
import words from '../../../../config/motivational-quotes.json';
```
This is a more standard filename convention.

ojeytonwilliams commented on client/src/utils/get-words.js:
```diff
@@ -1,4 +1,4 @@
-import words from './words.json';
+import words from '../../../config/motivationalQuotes.json';
```
```suggestion
import words from '../../../config/motivational-quotes.json';
```

ojeytonwilliams reviewed (changes requested):
Hi @rdm100 we're nearly there, but I noticed a few things we could clean up.

Ideally filenames should be lower case, so `motivationalQuotes.json` should become `motivational-quotes.json`.

`get-words.js` appears twice, so let's remove `client/src/templates/Challenges/utils/get-words.js` in favour of `client/src/utils/get-words.js`.  Also, `randomPhrase` and `randomVerb` are no longer used, so those functions and the corresponding parts of `motivational-quotes.json` (`phrases` and `verbs`) should go.

rdm100 commented:
@Sky020 I removed client/src/templates/Challenges/utils/get-words.js, but had to add it back in as the build failed because client/src/templates/Challenges/redux/current-challenge-saga.js uses it.

rdm100 committed:
Audited and Consolidated congratulatory quotes into 1 file and updated the paths to the files that use it.

rdm100 commented:
@Sky020 just realised that file should just reference the only get-words.js file remaining, so did that.

Sky020 commented:
That looks fine, so far. If you have done what _ojeytonwilliams_ suggested, then you can mark the request as _Resolved_. This lets him know you have made the changes, and he can review it again.

rdm100 commented:
@ojeytonwilliams resolved the issues you flagged I think, so it should be ready for review again.

ojeytonwilliams committed:
refactor: DRY out quote generation

ojeytonwilliams reviewed (approved):
Thanks for making those changes, @rdm100!  I made a mistake and missed some duplicated code in my first review, so I pushed a commit fixing that.

It looks good to me, now!

Manish-Giri reviewed (approved):
LGTM 💯

rdm100 commented:
cool, cheers everyone for answering any questions I had.

ojeytonwilliams commented:
No problem @rdm100, thanks for taking the time to do this.  It will definitely make maintaining this a lot easier.
